assignment3.txt

```diff
@@ -0,0 +1,26 @@
+Emma Ladouceur
+
+Assignment Answers
+
+
+1. I dont' think it is valid.
+	First after the %d what is the code printing? That means it should be preparing for an integer and there is nothing defined nor is there a variable name following in. 
+	Secondly, there is a semicolon before the i.
+
+
+2. 
+There should not be a semi colon after the first set of () because it will go straight to the {}.
```

Comment:
Oh, I forgot to note that the for loop also enters an infinite loop, so that would also be an issue!